sidebars.js

```diff
@@ -1250,7 +1250,16 @@ const sidebars = {
         "operations/ssl-zookeeper",
         "operations/startup-scripts",
         "operations/storing-data",
-        "operations/allocation-profiling",
+        {
+          type: "category",
+          label: "Allocation profiling",
+          collapsed: true,
+          collapsible: true,
+          link: { type: "doc", id: "operations/allocation-profiling" },
+          items: [
+            "operations/allocation-profiling-old",
+          ]
```

Comment:
```suggestion
          link: { type: "doc", id: "operations/allocation-profiling" },
          items: [
            "operations/allocation-profiling",
            "operations/allocation-profiling-old",
          ]
```

Otherwise only the old page shows up in the drop down items. I think users might not think to click the category to see the new page.

<img width="267" height="140" alt="Screenshot 2025-09-05 at 13 45 45" src="https://github.com/user-attachments/assets/994a74ca-75ac-4bc3-9354-8bd691a2a00f" />

Reply:
The new page will show as the default one. If they need the old page it's there as well below, think this is okay.
<img width="274" height="148" alt="Screenshot 2025-09-05 at 14 30 07" src="https://github.com/user-attachments/assets/d211d39a-16dc-46ff-93d0-79253ec3a2d8" />